tgui/packages/tgui/interfaces/Terminal.js

```diff
@@ -0,0 +1,80 @@
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+import { Box, Input, Button, Section } from 'tgui/components';
+import { ColorBox, Flex } from '../components';
+
+export const Terminal = (props, context) => {
+  const { act, data } = useBackend(context);
+  const peripherals = data.peripherals || [];
+  const { textInput } = props;
+
+  const {
+    fdisk,
+    idcard,
+    displayHTML,
+    TermActive,
+    windowName,
+    user,
+    fontColor,
+    bgColor,
+    diskdrive,
+  } = data;
+
+  return (
+    <Window
+      theme="retro-dark"
+      title={windowName}
+      width="380"
+      fontFamily="Consolas"
+      font-size="10pt"
+      height="350">
+      <Window.Content>
+        <Section backgroundColor="#0f0f0f" scrollable fill height="70%" >
+          <Box color={fontColor} backgroundColor={bgColor} minHeight="99%" maxHeight="500em">
+            <Box mx="1%"
+              fontFamily="Consolas"
+              fill
+              color={fontColor}
+              backgroundColor={bgColor}
+              dangerouslySetInnerHTML={{ __html: displayHTML }}
```

Comment:
Could you list specific examples of why this is required? We might be able to solve in another way.

Reply:
for example, all the computer code does manual bolding and italicizing in strings with html

Reply:
the var that the terminals use to display text is stored as a long list of HTML which is used for certain things like changing a line of text to red for error messages, and for making certain parts bold, such as the Thinktronics BIOS logo on startup.

example of the error text: 
![image](https://user-images.githubusercontent.com/64938519/169666542-77e14094-9c32-4873-9913-756d709868f1.png)
example of the bold Thinktronics logo:
![image](https://user-images.githubusercontent.com/64938519/169666571-85f4bd4a-bc51-4bd1-b386-af4ed9dfca9d.png)

Reply:
Hmmm. Makes sense. It's not ideal, but I guess we're already dumping raw HTML at the user so this doesn't expose any new issues.